src/lib/Microsoft.Health.Events/Telemetry/Exceptions/InvalidEventHubException.cs

```diff
@@ -0,0 +1,29 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using Microsoft.Health.Common.Telemetry;
+
+namespace Microsoft.Health.Events.Telemetry.Exceptions
+{
+    public sealed class InvalidEventHubException : IomtTelemetryFormattableException
+    {
+        public InvalidEventHubException(
+            string message,
+            Exception innerException,
+            string errorName)
+            : base(
+                  message,
+                  innerException,
+                  name: $"{ErrorType.EventHubError}{errorName}",
```

Comment:
nit: Can just use ErrType now so that if that changes for whatever reason, we won't have to remember to change here too.